backend/src/nodes/nodes/image_filter/blur_bilateral.py

```diff
@@ -0,0 +1,68 @@
+from __future__ import annotations
+
+import cv2
+import numpy as np
+
+from . import category as ImageFilterCategory
+from ...node_base import NodeBase
+from ...node_factory import NodeFactory
+from ...properties.inputs import ImageInput, NumberInput
+from ...properties.outputs import ImageOutput
+from ...utils.utils import get_h_w_c
+
+
+@NodeFactory.register("chainner:image:bilateral_blur")
+class BlurNode(NodeBase):
+    def __init__(self):
+        super().__init__()
+        self.description = "Apply surface/bilateral blur to an image."
+        self.inputs = [
+            ImageInput(),
+            NumberInput("Radius", controls_step=1),
```

Comment:
According to the docs, this is diameter, not radius.

Reply:
That would make sense why I had to use 12 to get similar results to paint.net's 6

Reply:
Should I just multiply the radius by 2 then? or should I change this to be diameter?